auslib/test/admin/views/test_dockerflow.py

```diff
@@ -0,0 +1,34 @@
+import mock
+
+from auslib.test.admin.views.base import ViewTest
+
+
+class TestDockerflowEndpoints(ViewTest):
+
+    def testVersion(self):
+        ret = self.client.get("/__version__")
+        self.assertEquals(ret.data, """
+{
+  "source":"https://github.com/mozilla/balrog",
+  "version":"1.0",
+  "commit":"abcdef123456"
+}
+""")
+
+    def testHeartbeat(self):
+        with mock.patch("auslib.global_state.dbo.rules.countRules") as cr:
+            ret = self.client.get("/__heartbeat__")
+            self.assertEqual(ret.status_code, 200)
+            self.assertEqual(cr.call_count, 1)
+
+    def testHeartbeatWithException(self):
+        with mock.patch("auslib.global_state.dbo.rules.countRules") as cr:
+            cr.side_effect = Exception("kabom!")
+            # Because there's no web server between us and the endpoint, we recieve
+            # the Exception directly instead of a 500 error
+            self.assertRaises(Exception, self.client.get, "/__heartbeat__")
```

Comment:
Will the actual endpoint go through the 500 handler and return <updates></updates> ? Does that matter to the monitoring system ?

s/kabom/kaboom/, or is that a reference I missed ?
s/recieve/receive/